nix/modules/blocksense/default.nix

```
@@ -0,0 +1,49 @@
+{
+  lib,
+  self,
+  inputs,
+  withSystem,
+  ...
+}:
+{
+  flake.nixosModules =
+    let
+      mkModule =
+        backend:
+        { config, pkgs, ... }:
+        let
+          cfg = config.services.blocksense;
+
+          inherit (withSystem pkgs.stdenv.hostPlatform.system (args: args)) inputs' self';
+
+          specialArgs = {
+            inherit
+              pkgs
+              self
+              self'
+              inputs
+              inputs'
+              cfg
+              lib
+              ;
+          };
+        in
+        {
+          options.services.blocksense = import ./module-opts specialArgs;
+
+          config.services.blocksense = {
+            sequencer.providers = lib.mkIf (cfg.anvil != { }) (
+              lib.mapAttrs (_name: value: {
+                "url" = lib.mkDefault "http://127.0.0.1:${toString value.port}";
```

Given that the port is a service option, why not make the ip/endpoint one as well, that defaults to http://127.0.0.1?

```suggestion
                "url" = lib.mkDefault "${value.ip}:${toString value.port}";
```

---

Unrelated to this PR but anvil in our case is always local